scipy/stats/tests/test_distributions.py

```diff
@@ -395,6 +395,49 @@ def test_pdf_R(self):
         vals_pdf = stats.norminvgauss.pdf(x_test, a=1, b=0.5)
         assert_allclose(vals_pdf, r_pdf, atol=1e-9)
 
+    @pytest.mark.parametrize('x, a, b, sf, rtol',
+                             [(-1, 1, 0, 0.8759652211005315, 1e-13),
+                              (25, 1, 0, 1.1318690184042579e-13, 1e-4),
+                              (1, 5, -1.5, 0.002066711134653577, 1e-12),
+                              (10, 5, -1.5, 2.308435233930669e-29, 1e-9)])
+    def test_sf_isf_mpmath(self, x, a, b, sf, rtol):
```

Comment:
maybe you can add a test to check that the _sf / _isf work for non-scalar input? you could reuse the same test cases, e.g.

sf([-1, 25], [1, 0], [0, 0])

Reply:
@chrisb83 added the requested test. I think there has been enough time for comments, so if this still looks good to you, feel free to merge!